### ISSUE TITLE
Improving UI in Hospital Login page

### ISSUE THREAD

**Issue by LunlunChin** [pranjay-poddar/Swasthify#95]
**Describe the Feature you want to end**
I would like to improve the UI on hospital login page

**Expected behavior**
I will remove the underline on the signup link & I will add a new remember me checkbox in the login form

**Screenshots (optional)**
![underline](https://user-images.githubusercontent.com/36696204/194705260-4a2c2d72-c1d7-4413-9205-a47721b17201.png)

![remember](https://user-images.githubusercontent.com/36696204/194705411-64280573-ee34-409d-90f0-71a9dd636b18.png)

**Comment by pranjay-poddar** [comment]
Yeah go ahead, but I would expect some more UI enhancements if possible.

**Comment by LunlunChin** [comment]
Yes sure, I will continue to work on the UI improvement!! Thanks for giving me this opportunity to contribute to open source!

**Comment by sumitbaroniya** [comment]
please assign me this issue.

**Comment by tannu610** [comment]
Hii @pranjay-poddar please assign me this issue i want to improve the login card 
Now sign up link is present at the too buttom of the page i want to add that link in the login card it self so it will look more better

![Screenshot (206)](https://user-images.githubusercontent.com/92167499/195553884-05e37f3f-d3df-4549-ae48-701aa6c8b79d.png)

https://user-images.githubusercontent.com/92167499/195550642-cfe18339-48c3-49b6-8b80-ebcf759b9029.jpg
